src/test/java/seedu/address/model/person/AddressTest.java

```diff
@@ -10,27 +10,27 @@
 
     @Test
     public void constructor_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> new Address(null));
+        assertThrows(NullPointerException.class, () -> new Id(null));
     }
 
     @Test
     public void constructor_invalidAddress_throwsIllegalArgumentException() {
         String invalidAddress = "";
-        assertThrows(IllegalArgumentException.class, () -> new Address(invalidAddress));
+        assertThrows(IllegalArgumentException.class, () -> new Id(invalidAddress));
     }
 
     @Test
     public void isValidAddress() {
         // null address
-        assertThrows(NullPointerException.class, () -> Address.isValidAddress(null));
+        assertThrows(NullPointerException.class, () -> Id.isValidId(null));
 
         // invalid addresses
-        assertFalse(Address.isValidAddress("")); // empty string
-        assertFalse(Address.isValidAddress(" ")); // spaces only
+        assertFalse(Id.isValidId("")); // empty string
+        assertFalse(Id.isValidId(" ")); // spaces only
 
         // valid addresses
-        assertTrue(Address.isValidAddress("Blk 456, Den Road, #01-355"));
-        assertTrue(Address.isValidAddress("-")); // one character
-        assertTrue(Address.isValidAddress("Leng Inc; 1234 Market St; San Francisco CA 2349879; USA")); // long address
+        assertTrue(Id.isValidId("A0123456H"));
+        assertTrue(Id.isValidId("A0123457H"));
+        assertTrue(Id.isValidId("A0123458H"));
```

Comment:
Good test cases for valid Patron Id and as LibTask progresses, we might want to add more test cases for staff member ids.
Refer to the following [link](https://libfaq.nus.edu.sg/faq/71298).
For example, ```assertTrue(Id.isValidId("012345"));```